### PR TITLE
USWDS-3676: Collection Component

### LINKED ISSUE #3676: Create Collection component
## Milestones

DESIGN ASSETS
- [x] 10% - Collect and analyze examples;  draft research insights and list of features for team/client feedback - on [Mural board](https://app.mural.co/t/flexion6734/m/flexion6734/1600718591376/0b4f8d81512a19034ca5fc1f1759c8913c20b359) - done Mon 11/2
- [x] 20% - Draft user story & clarify the problem(s) / user needs - on [Mural board](https://app.mural.co/t/flexion6734/m/flexion6734/1600718591376/0b4f8d81512a19034ca5fc1f1759c8913c20b359) - done Tue 11/3
- [x] 30% - Low-fi design review - done Tue 11/3
- [x] 40% - Get community feedback on initial, Hi-fi designs - done Tue 11/17
- [ ] 50% - Hi-fi Sketch assets review **- due Thu 12/3**
- [ ] 60% - Hi-fi Adobe XD assets review **- due Thu 12/3**
- [ ] 80% - Sketch and XD assets review w/ Dan + merged into master working files **- due Fri 12/4**
- [ ] 100% - PR approved by Dan **- due Fri 12/4**

GUIDANCE
- [x] 10% - Summarize insights from research (desk + interviews) - done Mon 11/2
- [x] 20% - Draft guidance review no. 1 (minimum 2 bullets each section) - done Thu 11/12
- [x] 40% - Get community feedback on draft guidance - done Tue 11/17
- [x] 50% - Draft guidance review no. 2 (accessibility focus) - done Tue 12/1
- [x] 60% - Draft guidance review no. 3 (w/ Dan) - done Tue 12/1
- [x] 80% - Add component to research report and slide deck **- due Thu 12/3**
- [ ] 100% - PR approved by Dan **- due Fri 12/4**

CODE
- [x] 10% - Code review no. 1 ([Federalist preview](https://federalist-3b6ba08e-0df4-44c9-ac73-6fc193b0e19c.app.cloud.gov/preview/uswds/uswds/accelerator/3676-news-feed-prototype/components/detail/featured-content.html)) - interaction design focus - done Tue 11/10
- [x] 20% - Code review no. 2 (Federalist preview) - visual design focus - done Fri 11/13
- [x] 40% - Get community feedback or conduct tests with users, round 1 - done Wed 11/18
- [x] 60% - Code review no. 3 (Federalist preview) - accessibility focus **- due Thu 12/3**
- [x] 80% - Get community feedback or conduct tests with users, round 2 **- due Thu 12/3**
- [ ] 100% - PR approved by Dan **- due Fri 12/4**

### PR DESCRIPTION
**Pull request by wbyoko**
## Description

Collection component 

closes #3676

[Federalist Preview](https://federalist-3b6ba08e-0df4-44c9-ac73-6fc193b0e19c.app.cloud.gov/preview/uswds/uswds/accelerator/3676-news-feed-prototype/components/detail/collection--default.html)

## Additional information

Screen Shot 2020-11-30 at 7 00 04 PM
Screen Shot 2020-11-30 at 7 00 13 PM
Screen Shot 2020-11-30 at 7 00 19 PM
Screen Shot 2020-11-30 at 7 00 26 PM

Before you hit Submit, make sure you’ve done whichever of these applies to you:

- [x] Follow the [18F Front End Coding Style Guide](https://pages.18f.gov/frontend/) and [Accessibility Guide](https://pages.18f.gov/accessibility/checklist/).
- [x] Run `npm test` and make sure the tests for the files you have changed have passed.
- [x] Run your code through [HTML_CodeSniffer](http://squizlabs.github.io/HTML_CodeSniffer/) and make sure it’s error free.
- [x] Title your pull request using this format: [Website] - [UI component]: Brief statement describing what this pull request solves.

### CODE REVIEW

**Review by 3DX8 — COMMENTED** [review]
I pulled your branch, ran it locally and tested a few things... Also tested in CrossBrowserTest.com... It's looking good to me @wbyoko nice work

File: src/components/13-collection/collection--fancy-date.njk [hunk 1]
```diff
@@ -0,0 +1,188 @@
...
+              </time>
+            </span>
+          </div>
+
+          <div class="usa-collection__meta-list">
+            <span class="usa-collection__meta-item usa-tag">
+              SBA
+            </span>
+            <span class="usa-collection__meta-item usa-tag">
+              Beta data
```
Comment (sknep) [inline src/components/13-collection/collection--fancy-date.njk:69]: Should this say "Meta data"?

File: src/components/13-collection/collection--fancy-date.njk [hunk 2]
```diff
@@ -0,0 +1,188 @@
...
+            </span>
+            <span class="usa-collection__meta-item">
+              <time datetime="2020-09-30T12:00:00+01:00">
+                September 30, 2020
+              </time>
+            </span>
+          </div>
+
+          <div class="usa-collection__meta-list">
+            <span class="usa-collection__meta-item usa-tag">
```
Comment (sknep) [inline src/components/13-collection/collection--fancy-date.njk:29]: I'm trying to figure out how non-visual experiences will read this markup... is there anything we can do to identify these as tags/labels? https://www.w3.org/WAI/WCAG21/Techniques/html/H49

Comment (sknep) [reply src/components/13-collection/collection--fancy-date.njk:29]: @jonadecker any ideas?

File: src/components/13-collection/collection--fancy-date.njk (hunk 1 above)
Comment (wbyoko) [reply src/components/13-collection/collection--fancy-date.njk:69]: That was the content from the examples but I was wondering the same so I'll update it. Thanks!

File: src/components/13-collection/collection--fancy-date.njk (hunk 2 above)
Comment (jonadecker) [reply src/components/13-collection/collection--fancy-date.njk:29]: I'm wondering if it wouldn't be more appropriate to gather the tags as an unordered list @sknep @wbyoko . The preference always goes to semantic HTML vs. bandaid.

Comment (thisisdano) [reply src/components/13-collection/collection--fancy-date.njk]: DONE

**Review by thisisdano — APPROVED** [review]

**Pull request merged**